server/app/assets/stylesheets/styles.css

```diff
@@ -33,4 +33,14 @@
   .cf-accordion-visible > .cf-accordion-content {
     @apply h-auto mt-2 mb-1;
   }
+
+  /* Remove number steppers (https://www.w3schools.com/howto/howto_css_hide_arrow_number.asp) */
+  input::-webkit-outer-spin-button,
```

Comment:
This removes number steppers globally. Is that desired, or do we want to remove number steppers for the numerical type of question specifically?

If it's the latter, we might need to specify a CSS class.

Reply:
Good callout! I updated the PR description to call out that this will be a global change since the original issue referenced both the admin and applicant-facing UI.

Should we want to reintroduce this for a specific input at a later point, we can introduce the CSS class when needed.